node21-expressjs/app/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
+const port = 3000
```

Comment:
`3000` or `8080`?

Reply:
I see in the test you used 3000, so I guess one for sure needs changing

Reply:
@craciunoiuc , it's `3000`.